test_scripts/Polices/build_options/ATF_P_Register_NewApp_not_exist_inLocalPT_FinishPTU_PROPRIETARY.lua

```diff
@@ -80,6 +80,12 @@ function Test:Precondition_PolicyUpdateStarted_ForDefaultApplication()
   EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "PROPRIETARY" })
 end
 
+function Test:Precondition_ActivateApp()
+  local RequestId = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = self.applications[config.application1.registerAppInterfaceParams.appName]})
+  EXPECT_HMIRESPONSE(RequestId)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
```

Comment:
@dboltovskyi, you are right, but in case you have both applications in NONE, SDL will switch between one of them, so you can't be sure that OnSystemRequest will be sent to app1(line 126).
When first app has HMI level different that NONE and second is in NONE, SDL for sure will choose App1.